File: Java/Desafio2 - GFT/src/model/Carro.java
package model;


public class Carro extends Veiculo {
	
	private String marca;
	private int portas;
	private int ano;
			
	public String getMarca() {
		return marca;
	}
	public void setMarca(String marca) {
		this.marca = marca;
	}
	public int getPortas() {
		return portas;
	}
	public void setPortas(int portas) {
		this.portas = portas;
	}
	public int getAno() {
		return ano;
	}
	public void setAno(int ano) {
		this.ano = ano;
	}
	
	public Carro(String modelo, int velocidade, int passageiros, int combustivel, String marca, int portas, int ano) {
		super(modelo, velocidade, passageiros, combustivel);
		setMarca(marca);
		setPortas(portas);
		setAno(ano);
	}
	
}

File: C-Sharp/Desafio02/Desafio02/com.gft.model/Carro.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Desafio02.com.gft.model
{
    class Carro:Veiculo 
    {
        private String marca;
        private int portas;
        private int ano;

        public Carro(string modelo, int velocidade, int passageiros, int combustivel, string marca, int portas, int ano):base(modelo, velocidade, passageiros, combustivel)
        {
            this.marca = marca;
            this.portas = portas;
            this.ano = ano;
        }

        public string Marca { get => marca; set => marca = value; }
        public int Portas { get => portas; set => portas = value; }
        public int Ano { get => ano; set => ano = value; }
    }
}
